plugin/pkg/shoot/nldns/admission.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Nit: Can you please rename the package from `nldns` to `nodelocaldns` so that we don't have magic abbreviations?

Reply:
Renamed it accordingly